client/performance-profiler/components/insights-section/style.scss

```diff
@@ -1,10 +1,13 @@
+@import "@automattic/color-studio/dist/color-variables";
 @import "@wordpress/base-styles/breakpoints";
 
-$blueberry-color: #3858e9;
```

Comment:
We should avoid duplicating color variables like this as it make it harder to manage colors long-term. Please pull colors from `@automattic/color-studio/dist/color-variables` instead.